pages/sync/bitbucket.en.mdx

```diff
@@ -0,0 +1,99 @@
+# Bitbucket sync setup guide
+[Bitbucket](https://bitbucket.org/product) is a Git-based source code repository hosting service popular among teams using Atlassian tools. 
+
+You can use the Tokens Studio plugin native integration with Bitbucket to sync your design tokens to a repository of your choice. 
+
+We support bi-directional sync, meaning you can use the plugin to: 
+- `push` JSON files full of design tokens to Bitbucket
+- `pull` the tokens stored in Bitbucket into any Figma file
+
+This means the design tokens living in code are the source of truth for our design decisions, which can be shared between design and development teams. 
+
+This doc outlines how to set up a Bitbucket repository and add it as a `Sync provider` in the plugin. 
+## How it works
+- Set up a `repository` and `personal access token` in Bitbucket.
+- Configure `Bitbucket as a sync provider` within the Tokens Studio plugin.
+- Seamlessly `sync your design tokens` between Figma and Bitbucket to maintain consistency across design and development workflows.
+
+## Bitbucket setup instructions
+If you haven't already, sign up for a Bitbucket account at https://bitbucket.org/
+
+### 1. Create a new repository
+Sign into your account and select the `Create` button, and select `Repository`.
+- Choose a specific and memorable `repository name` which will be used in the plugin as the destination for your design tokens. 
+- Review the options, [learn more about those in the Bitbucket docs.](https://support.atlassian.com/bitbucket-cloud/docs/create-a-git-repository/)
+	- `Add a README file` needs to be checked
+		- This is mandatory because the plugin can not sync to an empty repository.
+	- The other options are up to you.
+
+You are now looking at your new repository! Well done! 
+- `Save the URL of the repository` somewhere for safekeeping and plugin configuration later. 
+
+### 2. Generate an app password
+Navigate to your Bitbucket account settings.
+- Locate the `Personal settings` section, select `Personal Bitbucket settings`
+- Select `App passwords`.
+- Select `Create app passwords`
+- Add a `Name` of what the app password is for
+	- Example: `test-token repo sync to tokens studio`
+- Select the `permissions` for app passwords
+	- `Account` = `Read`
+	- `Repositories` = `Write`
+- Select the `Create` button
+- The page will display the `New app password dialog`
+
+ `Copy the generated token` and store it securely, as you will need it to configure the connection with the Tokens Studio plugin.
```

Comment:
```suggestion
**Copy the generated token** and store it securely, as you will need it to configure the connection with the Tokens Studio plugin.
```

Can we find some consistency in what we mark with backticks? Usually I do this for stuff that has a specifi meaning, here it seems its just to emphasize. Let's use bold for stuff like this instead.

Reply:
Yes! Ideally this happens with the Editor. This isn't my strength and Ajit is aware and ready to help with these consistency checks.